src/lib.rs

```diff
@@ -6,4 +6,4 @@ pub mod roblox;
 #[cfg(test)]
 mod tests;
 
-pub use crate::lune::{Lune, LuneError};
+pub use crate::lune::{lua, Lune, LuneError};
```

Comment:
I don't think we should export the entire `lua` module, this would more or less export the entire Lune runtime and builtin libraries and if we change any of it we break semver compatibility for lib consumers due to it being exported.

```suggestion
pub use crate::lune::{Lune, LuneError};
```

Reply:
How exactly do I go about using `lua::formatting::*` in repl.rs?

Reply:
My bad, I didn't notice that we were actually dealing with a `LuneError` and not a `LuaError`, you can actually remove the dependency on the formatting module & function completely here since `LuneError` is just an opaque and already formatted `LuaError`, and print it out directly:
```rs
eprintln!("{err}") // err is a LuneError and implements Display
```